src/components/templates/PosterHall/components/PosterPreview/PosterPreview.tsx

```diff
@@ -46,6 +47,8 @@ export const PosterPreview: React.FC<PosterPreviewProps> = ({
 
       <div className="PosterPreview__categories">{renderedCategories}</div>
 
+      <img src={thumbnailUrl} alt="Thumbnail" />
```

Comment:
could this be made conditional on availability of the thumbnail and replace thumbnailUrl with some placeholder (e.g. just stating "No thumbnail") better than default "no image" icon etc I see currently:

![image](https://user-images.githubusercontent.com/39889/120831534-a0fbf880-c52d-11eb-99a9-660cf1fd88c8.png)

Reply:
Just an idea, but perhaps if the thumbnail is missing, we could simply display a colored rectangle, where is the color is randomly selected from a few options?